acceptance/apps_test.go

```diff
@@ -447,6 +448,31 @@ spec:
 
 	})
 
+	When("pushing as a stateful app", func() {
+		AfterEach(func() {
+			env.DeleteApp(appName)
+		})
+
+		It("pushes successfully", func() {
+			pushLog, err := env.EpinioPush("../assets/sample-app",
+				appName,
+				"--app-chart", "standard-stateful",
+				"--name", appName)
+			Expect(err).ToNot(HaveOccurred(), pushLog)
+
+			Eventually(func() string {
+				out, err := env.Epinio("", "app", "list")
+				Expect(err).ToNot(HaveOccurred(), out)
+				return out
+			}, "5m").Should(
+				HaveATable(
+					WithHeaders("NAME", "CREATED", "STATUS", "ROUTES", "CONFIGURATIONS", "STATUS DETAILS"),
+					WithRow(appName, WithDate(), "1/1", appName+".*", "", ""),
```

Comment:
Should we check that what was deployed is really StatefulSets and not Deployments? Just in case `standard-stateful` flag has no effect.

Reply:
Agreed, would be better. Working on that now.

Reply:
Done, commit 2bcab59812aeb6c634f7576252c9d9394de01639